mafiasi/dashboard/static/css/dashboard-christmas.css

```diff
@@ -1,7 +1,11 @@
-#santa-hat-services-deco {
+#dashboard > h2 {
     position: relative;
-    top: -56px;
-    left: -6px;
+}
+
+#santa-hat-services-deco {
     width: 24px;
     height: 24px;
+    position: absolute;
+    top: -13px;
+    left: -6px;
```

Comment:
By using `position: absolute;` we remove the santa-hat from the document flow - this was causing the additional margin.